### ISSUE TITLE
One Piece: Grand Adventure table has lines through the bottom of it

### ISSUE THREAD

**Issue by JBoughter** [Tinob/Ishiiruka#37]
This only appears in DirectX 11/12/OpenGL, and is not in DirectX 9. This appears on both stable and unstable, and does not appear in 4.0-8743 either.

Here's two screenshots I took to describe it. It occurs on "Arlong Park" on the Banquet night version. The first is of what it appears with Direct3D11 on 4.0-8743, the second being Ishiiruka 524 on DirectX 12.

#### 4.0-8743
![4.0-8743](http://i.imgur.com/H5wKr4U.png)

#### Ishiiruka Unstable (524)
![Ishiiruka 524](http://i.imgur.com/znSCQgB.png)

I fiddled with nearly every setting I could on the DirectX 12 renderer, but none of the settings fixed the table having lines. It seems like this may be an issue that settings can't resolve.

**Comment by Tinob** [comment]
the other backends have the same issue?

**Comment by JBoughter** [comment]
Yep. It occurs in all of them except for DirectX 9. Master completely doesn't have the issue.
